Daily PPA is Failing on Xenial

#5545 breaks on old SDL versions

[Sample Buildlog](https://launchpadlibrarian.net/622054583/buildlog_ubuntu-xenial-amd64.widelands_2%3A1.1~bzr10377-202209061446~ubuntu16.04.1_BUILDING.txt.gz)

```
/<<PKGBUILDDIR>>/src/logic/map_objects/world/terrain_description.cc: In member function ‘void Widelands::TerrainDescription::replace_textures(const LuaTable&)’:
/<<PKGBUILDDIR>>/src/logic/map_objects/world/terrain_description.cc:194:67: error: ‘SDL_PIXELFORMAT_RGBA32’ was not declared in this scope
    SDL_Surface* converted = SDL_ConvertSurfaceFormat(sdl_surface, SDL_PIXELFORMAT_RGBA32, 0);
                                                                   ^
```

--- a/src/logic/map_objects/world/terrain_description.cc
+++ b/src/logic/map_objects/world/terrain_description.cc
@@ -191,7 +191,7 @@ void TerrainDescription::replace_textures(const LuaTable& table) {
 		// uint32, so let's not risk endianness issues, when 24 bits would be better off with
 		// conversion anyway
 		if (sdl_surface->format->BitsPerPixel != 32) {
-			SDL_Surface* converted = SDL_ConvertSurfaceFormat(sdl_surface, SDL_PIXELFORMAT_RGBA32, 0);
+			SDL_Surface* converted = SDL_ConvertSurfaceFormat(sdl_surface, SDL_PIXELFORMAT_RGBA8888, 0);
 			SDL_FreeSurface(sdl_surface);
 			if (converted == nullptr) {
 				throw WLWarning("", "Could not convert texture to 32bit RGB");

--- a/src/logic/map_objects/world/terrain_description.cc
+++ b/src/logic/map_objects/world/terrain_description.cc
@@ -191,7 +191,8 @@ void TerrainDescription::replace_textures(const LuaTable& table) {
 		// uint32, so let's not risk endianness issues, when 24 bits would be better off with
 		// conversion anyway
 		if (sdl_surface->format->BitsPerPixel != 32) {
-			SDL_Surface* converted = SDL_ConvertSurfaceFormat(sdl_surface, SDL_PIXELFORMAT_RGBA8888, 0);
+			SDL_Surface* converted =
+			   SDL_ConvertSurfaceFormat(sdl_surface, SDL_PIXELFORMAT_RGBA8888, 0);
 			SDL_FreeSurface(sdl_surface);
 			if (converted == nullptr) {
 				throw WLWarning("", "Could not convert texture to 32bit RGB");

